### PR TITLE
edbrowse 3.8.9 (new formula)

### PR DESCRIPTION
**Pull request by n0ot**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by n0ot** [comment]
brew audit fails because I'm using depends_on "curl". I actually need the version of curl from Homebrew, since Edbrowse requires it to be compiled with OpenSSL, not GnuTLS. How can I resolve this?

**Comment by p-linnane** [comment]
It's an audit we have since we prefer to use what is built-in on macOS. We can bypass it in cases like this where brewed `curl` is required.

You will also need to squash your commits into one with the message `edbrowse 3.8.9 (new formula)`.

File: Formula/e/edbrowse.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  depends_on "readline"
+  depends_on "unixodbc"
+
+  def install
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["curl"].opt_lib/"pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["openssl@3"].opt_lib/"pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["pcre2"].opt_lib/"pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["readline"].opt_lib/"pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["unixodbc"].opt_lib/"pkgconfig"
+
```
[the remark this answers is not in the document]
Comment (n0ot) [reply Formula/e/edbrowse.rb]: Interesting. It is when I manually install the dependencies on my system and then build edbrowse from source. I assume libs like curl, openssl, and readline, that are not normally linked are in the installation environment?

**Review by p-linnane — APPROVED** [review]
Thanks for your contribution to Homebrew @n0ot!

**Comment by p-linnane** [comment]
Actually need to rerun the jobs one more time. GitHub Actions is having some issues it seems.

**Pull request merged**